docs/index.rst

```diff
@@ -924,6 +926,37 @@ The `Confluent REST Proxy <https://docs.confluent.io/current/kafka-rest/docs/ind
 
        docker-compose exec restproxy curl -X DELETE -H "Content-Type: application/vnd.kafka.v2+json" --cert /etc/kafka/secrets/restproxy.certificate.pem --key /etc/kafka/secrets/restproxy.key --tlsv1.2 --cacert /etc/kafka/secrets/snakeoil-ca-1.crt -u appSA:appSA https://restproxy:8086/consumers/my_avro_consumer/instances/my_consumer_instance
 
+#. Now you will use the |crest| that is embedded on the |ak| brokers.  Create a role binding for the client to be granted ``ResourceOwner`` role for the topic ``dev_users``.
+
+   .. sourcecode:: bash
+
+      # First get the KAFKA_CLUSTER_ID
+      KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v3/clusters | jq ".data[0].cluster_id")
+
+      # Then create the role binding for the topic ``dev_users``
+      docker-compose exec tools bash -c "confluent iam rolebinding create \
+          --principal User:appSA \
+          --role ResourceOwner \
+          --resource Topic:usersq \
+          --kafka-cluster-id $KAFKA_CLUSTER_ID"
+
+#. Create the topic ``dev_users`` with the embedded |crest|.
+
+   .. sourcecode:: bash
+
+      # First get the KAFKA_CLUSTER_ID
+      KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v3/clusters | jq ".data[0].cluster_id")
+
+      docker-compose exec restproxy curl -X POST -H "Content-Type: application/json" -H "accept: application/json" --cert /etc/kafka/secrets/restproxy.certificate.pem --key /etc/kafka/secrets/restproxy.key --tlsv1.2 --cacert /etc/kafka/secrets/snakeoil-ca-1.crt -u appSA:appS "http://localhost:8082/v3/clusters/${KAFKA_CLUSTER_ID}/topics" -d "{\"topic_name\":\"dev_users\",\"partitions_count\":64,\"replication_factor\":2,\"configs\":[{\"name\":\"cleanup.policy\",\"value\":\"compact\"},{\"name\":\"compression.type\",\"value\":\"gzip\"}]}"
```

Comment:
This is hitting port 8082, isn't that contradictory with the earlier statement:

```
While in embedded mode |crest| listens for requests on http://kafka1:8091 and http://kafka2:8092 providing only the :ref:`rest-proxy-v3` are supported this time.
```

Reply:
Right you are. Does it just make sense to use the v3 APIs on the standalone leaving it up to the reader to try v3 apis against the embedded instance at their leisure.

Reply:
IIUC v3 doesn't support produce/consume, so the standalone can't use v3?  If that is correct, then I would suggest:

a) standalone uses v1 with produce/consume -- requires no change to existing docs
b) ERP uses v3 with just topic creation

Reply:
a) standalone uses V2 with produce/consume (V1 is gone in 6.0)

Reply:
@rigelbm correction, cp-demo example is already using v2 for the standalone (not v1).  If 6.0 supports v2 and v3, then this PR can optionally update all the commands to use v3 syntax for standalone.

Still open issue is reconciling the ports: why is the ERP example using 8082 when the added writing says 8091 and 8092

Reply:
It's a typo again my suggestion is to not mix the two in the examples but instead mention that ERP listens on 809[1-2] in the intro if you want to execute v3 requests against it.

Reply:
My strong leaning is that if this PR adds ERP to the configuration, then the PR should also add an exercise to execute statements to the ERP.

Reply:
Both examples point to kafka1:8091 now